### DISCUSSION TITLE
Merging reports and coverage when sharding?

### DISCUSSION THREAD

**Discussion by joshdholtz** [vitest-dev/vitest#4227] [Q&A] [answered]
I've recently been helping my team migrate from a Create React App over to Vite. One of the things that we wanted to take advantage of was speeding up our CI process by sharding. We were able to do all that (thanks to all of the amazing work on Vite and Vitest 🙏) but I know have a question about sharding.

Is there a built-in way or a standard process for merging together test reports and coverage reports from sharded test runs?

I saw that Playwrite has some test sharding doc about merging reports [here](https://playwright.dev/docs/test-sharding) and I was wondering if Vitest has something similar that I should be using? Or is this something that needs to be built yet?

Thanks!

**Comment by sheremet-va** [comment, marked as answer]
Unfortunately, we don't have anything for reports merging at the moment.

**Reply by joshdholtz** [reply]
No worries! And I appreciate that quick reply 😊 Is this something that is on the roadmap and/or something that you would be open to a contribution to if I can sneak it in my schedule?

**Reply by sheremet-va** [reply]
Not on the roadmap to be honest, but we are open to contributions! You can open an issue and I will add a label on it.

**Reply by joshdholtz** [reply]
Aye aye, will do! I have never contributed to any Javascript tool so kind of excited if I'm able to do it 😛
